assets/css/button.css

```diff
@@ -6,7 +6,7 @@ a.bulib-btn, button.bulib-btn {
   text-align: center;
   margin: var(--padding-small, 5px);
   padding: var(--padding-small, 5px) var(--padding-medium, 10px);
-  color: var(--color-button-text, white);
+  color: var(--color-button-text, white) !important;
```

Comment:
🐞 have to add a corresponding `!important` to `color` for secondary. currently displaying whitesmoke on white!